src/convertToObject.js

```diff
@@ -5,8 +5,27 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
+function convertToObject(styleString) {
+  const stylesObject = {};
+  const cleanedStyleString = styleString.trim();
+  const declarations = cleanedStyleString.split(';');
+
+  declarations.forEach((declaration) => {
+    if (declaration.includes(':')) {
+      const [property, value] = declaration.split(':');
+
+      if (property && value) {
+        const trimmedProperty = property.trim();
+        const trimmedValue = value.trim();
+
+        if (trimmedProperty && trimmedValue) {
+          stylesObject[trimmedProperty] = trimmedValue;
+        }
+      }
```

Comment:
The current check `if (property && value)` ensures that both property and value exist before trimming. However, after trimming, either could still be an empty string. Consider updating the condition to check `if (trimmedProperty && trimmedValue)` to ensure both are non-empty after trimming.